### PR TITLE
[Bach]: `NULLS LAST` by default in Order By Clause

### PR DESCRIPTION
**Pull request by KathiaBarahona**
Pandas by default will sort values with nulls last (Postgres too). We should follow the same behavior for all engines.

### CODE REVIEW

File: bach/bach/partitioning.py [hunk 1]
```diff
@@ -336,6 +336,7 @@ class that would alter it returns a fresh copy. It can be reused many time if th
     def __init__(self,
                  group_by_columns: List['Series'],
                  order_by: List[SortColumn],
+                 nulls_last: bool = False,
```
Comment (KathiaBarahona) [inline bach/bach/partitioning.py:339]: cannot make it default, most window functions for BQ started failing. We should revisit this later

**Review by thijs-obj — COMMENTED** [review]
small comments

File: bach/tests/functional/bach/test_df_sort_values.py [hunk 2]
```diff
@@ -67,3 +95,19 @@ def test_sort_values_parameters(engine):
...
+
+
+@pytest.mark.parametrize(
+    "ascending",  [
+        *[list(perm) for perm in itertools.permutations([True, False, False])],
+        *[list(perm) for perm in itertools.permutations([True, True, False])],
+        [True, True, True],
+        [False, False, False],
+    ]
+)
```
[the remark this answers is not in the document]
Comment (KathiaBarahona) [reply bach/tests/functional/bach/test_df_sort_values.py:107]: oh, you're right

[the remark this answers is not in the document]
Comment (KathiaBarahona) [reply bach/tests/functional/bach/test_df_sort_values.py]: Fixed here https://github.com/objectiv/objectiv-analytics/pull/832/commits/cedfbf50dce6be84572ec4fe7dbf11c75262dba8

**Pull request merged**